features/guidance/bicycle-sliproads.feature

```diff
@@ -24,9 +24,9 @@ Feature: Bike - Mode flag
         And the ways
             | nodes | highway   | name   | oneway:bicycle |
             | abcd  | cycleway  | street |                |
-            | eb    | path      |        | yes            |
+            | be    | path      |        | yes            |
```

Comment:
was there a reason the slipway was oneone in the opposite direction? due to small changes in the speed on pushing bikes, the test was now failing. swtiching the oneway direction made the tests pass again